.buildkite/pipeline.serverless.yml

```diff
@@ -0,0 +1,82 @@
+env:
+  SETUP_GVM_VERSION: 'v0.5.2' # https://github.com/andrewkroh/gvm/issues/44#issuecomment-1013231151
+  DOCKER_COMPOSE_VERSION: "v2.24.1"
+  DOCKER_VERSION: "26.1.0"
+  KIND_VERSION: 'v0.20.0'
+  K8S_VERSION: 'v1.29.0'
+  LINUX_AGENT_IMAGE: "golang:${GO_VERSION}"
+  GH_CLI_VERSION: "2.29.0"
+  # Elastic package settings
+  # Manage docker output/logs
+  ELASTIC_PACKAGE_COMPOSE_DISABLE_VERBOSE_OUTPUT: "true"
+  # Default license to use by `elastic-package build`
+  ELASTIC_PACKAGE_REPOSITORY_LICENSE: "licenses/Elastic-2.0.txt"
+  # Link definitions path (full path to be set in the corresponding step)
+  ELASTIC_PACKAGE_LINKS_FILE_PATH: "links_table.yml"
+  # Disable comparison of results in pipeline tests to avoid errors related to GeoIP fields
+  ELASTIC_PACKAGE_SERVERLESS_PIPELINE_TEST_DISABLE_COMPARE_RESULTS: "true"
+
+steps:
+  - input: "Input values for the variables"
+    key: "input-variables"
+    fields:
+    - select: "SERVERLESS_PROJECT"
+      key: "SERVERLESS_PROJECT"
+      options:
+        - label: "observability"
+          value: "observability"
+        - label: "security"
+          value: "security"
+      default: "observability"
+    if: "build.source == 'ui'"
+
+  - wait: ~
+    if: "build.source == 'ui'"
+    allow_dependency_failure: false
+
+  - label: ":go: Run check-static"
+    key: check-static
+    command: "make check-static"
+    agents:
+      image: "${LINUX_AGENT_IMAGE}"
+      cpu: "8"
+      memory: "4G"
+
+  - label: ":go: :linux: Run unit tests"
```

Comment:
Skipped in this pipeline windows tests. They should already be run by the PR pipeline or by the push to the main branch.

Reply:
Umm, and do we need any unit tests here? :thinking: We could run only the integration tests for serverless.

Reply:
Ok, I'll remove this too :+1: 
These unit tests are going to be run in each PR , so it's not needed.

Reply:
Do you mean to remove also the step related to `make check-static`? @jsoriano

Reply:
Yep, we could also remove this step if we are not going to use them to block the execution of integration tests.

Reply:
I would keep `check-static` step, that will ensure that `elastic-package` can be compiled without errors.

Reply:
> Yep, we could also remove this step if we are not going to use them to block the execution of integration tests.

The script that starts the Elastic serverless project already performs a `make install` command https://github.com/elastic/elastic-package/pull/1807/files#diff-6b482aae5cb3ce434c95fe0f9238d6c8ed14a602031d6473ae03373f78ea937fR52

If there is any error there, that script should fail. So there should not be needed to keep `check-static` step.

I'll remove that step too.